packages/fast-animation/lib/animateGroup/index.ts

```diff
@@ -77,8 +77,13 @@ class AnimateGroup {
     /**
      * Returns the cumulative time it will take to complete an animation
      */
-    private getAnimationDuration(effectTiming: AnimationEffectTiming): number {
-        return (effectTiming.delay || 0) + (effectTiming.duration || 0);
+    private getAnimationDuration(effectTiming: EffectTiming): number {
+        const duration: string | number = effectTiming.duration;
```

Comment:
isn't this always a number because the enum is always a number for duration?

Reply:
Internally we use `number` so we can do maths, however TypeScript defines `EffectTiming.duration` as either a string or a number, so we need to ensure it's a number and parse it if it isn't.